web_widget_image_webcam/__manifest__.py

```diff
@@ -1,15 +1,13 @@
-# -*- coding: utf-8 -*-
-# Copyright 2016 Siddharth Bhalgami <siddharth.bhalgami@techreceptives.com>
+# Copyright (C) 2019-Today: Druidoo (<https://www.druidoo.io>)
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
 {
     "name": "Web Widget - Image WebCam",
     "summary": "Allows to take image with WebCam",
-    "version": "10.0.1.0.0",
+    "version": "12.0.1.0.0",
     "category": "web",
-    "website": "https://www.techreceptives.com",
-    "author": "Tech Receptives, "
-              "Odoo Community Association (OCA), "
-              "Kaushal Prajapati",
+    "website": "https://github.com/OCA/web",
+    "author": "Druidoo, "
+              "Odoo Community Association (OCA)",
```

Comment:
Do not change previous authors, and don't add druidoo as an author because we didn't create the module